pychunkedgraph/app/segmentation/v1/routes.py

```diff
@@ -204,6 +204,19 @@ def change_log_full(table_id):
     return output
 
 
+@bp.route("/table/<table_id>/tabular_change_log_recent", methods=["GET"])
+@auth_requires_permission("view")
```

Comment:
This is a quite expensive endpoint and I would like to limit it for now to certain users "view" and "edit" are too low